### PR TITLE
Add 'first' to service_entry.

### PR DESCRIPTION
**Pull request by bodymindarts**
Closes https://github.com/habitat-sh/habitat/issues/2004

Signed-off-by: Justin Carter <justin@starkandwayne.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/sup/src/manager/service/config.rs [hunk 1]
```diff
@@ -317,12 +317,18 @@ fn service_entry(census: &Census) -> toml::value::Table {
         .map(|ce| toml::Value::try_from(ce).expect("Can't convert into TOML Value"));
     let mut members: Vec<toml::Value> = Vec::new();
     let mut member_id = toml::value::Table::new();
-    for (sg, ce) in census.iter() {
-        members.push(toml::Value::try_from(ce).expect("Can't convert into TOML Value"));
-        member_id.insert(format!("{}", sg),
+    let mut first: bool = false;
```
Comment (mwrock) [inline components/sup/src/manager/service/config.rs]: did you mean to initialize to `true`?

Comment (bodymindarts) [inline components/sup/src/manager/service/config.rs]: Whoops

**Comment by bodymindarts** [comment]
@mwrock thanks for the review. Fixed the initialization.

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/215973470).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/215973470).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**